package.json

```diff
@@ -225,22 +225,30 @@
     "cover-integration": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/integration/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
     "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
-    "----------------------------------------- local test dbs ------------------------------------------": "",
+    "----------------------------------------- bulk tests ------------------------------------------": "",
+    "start-all": "grep 'bash dev' package.json | grep start | grep -vP \"(all)\" | cut -d'\"' -f4 | sort | xargs -n1 | grep dev | xargs -n1 bash",
+    "test-all": "grep 'test-' package.json | grep -vP \"(unit|integration|smoke|typings)\" | cut -d'\"' -f4 | xargs -n4 yarn ",
+    "stop-all": "grep 'bash dev' package.json | grep stop | grep -vP \"(all)\" | cut -d'\"' -f4 | sort | xargs -n1 | grep dev | xargs -n1 bash",
```

Comment:
can you move this to a `stop-all.sh` script in `/dev` and make this call it?

Reply:
Yes, if we're keeping these it might be better to move them all to `/dev`